### DISCUSSION TITLE
Artisan with Proaster

### DISCUSSION THREAD

**Discussion by BrooksTek** [artisan-roaster-scope/artisan#601] [Q&A] [answered]
Hi, I'm trying to config artisan with my roaster Machine Proaster, but it doesn't show temperature same as roaster Machine.
or I'm not config the right way. 

![photo_2021-04-07 14 35 16](https://user-images.githubusercontent.com/82081147/113828362-7f40f800-97ae-11eb-8b10-f4e43e1b9a2f.jpeg)
![photo_2021-04-07 14 35 09](https://user-images.githubusercontent.com/82081147/113828375-82d47f00-97ae-11eb-82bd-c45e53f1359b.jpeg)

**Comment by roasterdave** [comment, marked as answer]
Thanks for the photos.  Unfortunately it is not possible to tell if there are any decimal points lit on the Proaster temperature displays.  

If they are all without decimals there may be an unnecessary scaling in the Artisan setup.  Go to Config> Port> Modbus.  The "Divider" field can be configured to divide the data returned by the MODBUS device by 1/10 or 1/100 or not at all if left blank.  Try changing the divider value to Blank to see if that corrects the scaling for you.

Please report back if that solves it or not.

**Reply by BrooksTek** [reply]
Thank you for kind support. I did ready as you suggest, but it still doesn't work.

**Reply by roasterdave** [reply]
What are the temperatures displayed on the Proaster?  I cannot distinguish if there are decimal points from the photo and do not want to guess wrong.

**Reply by BrooksTek** [reply]
Like the picture I showed you upper, when the air 1 temperature to 321c and bean temperature 122 in the artisan show ET 32.1c & BT 12.2c. and below I tried to change device by 1/10 to 1/100 like what you guild me but still the same.
<img width="1319" alt="Screen Shot 2021-04-12 at 1 59 10 PM" src="https://user-images.githubusercontent.com/82081147/114355438-ab3aef80-9b99-11eb-9f25-24126c9de0d9.png">

**Reply by CarefreeBuzzBuzz** [reply]
Did you try it with the Divider field blank as suggested?

**Reply by AJGDD** [reply]
Dont divide the value, divider means ..... 
TK4H send the number (PV) 3225 ----(divide 1/10)---->  artisan show 322.5  
ooooor download DAQMaster and configure your controller with 1 decimal place
